gapic/src/main/com/google/gapid/util/Messages.java

```diff
@@ -48,6 +48,7 @@
   public static final String MEMORY_STRUCT_TAB_TEXT = "Struct";
   public static final String CAPTURE_TRACE_GRAPHICS = "Capture Graphics Trace";
   public static final String CAPTURE_TRACE_PERFETTO = "Capture System Profile";
+  public static final String CAPTURE_TRACE_SELECT = "Capture: Select Trace Type";
```

Comment:
I don't think this really makes sense as a title. Maybe use `Capture Trace` instead? I don't want to bikeshed this, so...lgtm.

Reply:
Updated!